.github/PULL_REQUEST_TEMPLATE/develop.md

```diff
@@ -0,0 +1,33 @@
+# Pull Request Details
+<!--- The title of the PR should summarize the change implemented. -->
+
+## Description
+<!--- Provide a general summary of your changes in the title above -->
+<!--- Why is this change required? What problem does it solve? -->
+
+## Related Issue
+<!--- Refer to any related issues here -->
+<!--- If this PR fully addresses an issue, please say "Fixes #1234", -->
+<!--- as this will allow Github to automatically close the related Issue -->
+
+## Which does this affect?
+<!--- Include areas/subsystems that are affected and some details on what -->
+<!--- areas these changes affect. -->
+
+## Testing Done
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you -->
+<!--- ran to see how your change affects other areas of the code, -->
+<!--- etc. Then, include justifications for how your tests -->
+<!--- demonstrate those affects. -->
+
+## Checklist
+
+<!--- Go over all the following points, and put an `x` in all the
+<!--- boxes that apply. Note that some of these may not be valid -->
+<!--- for all PRs. -->
+
+- [ ] My code follows the code style of this project.
+- [ ] My code has been tested on the Robot OR includes simulation code to prove functionality.
+- [ ] I have updated the documentation accordingly.
+- [ ] I have added tests to cover my changes, and all previous tests pass.
```

Comment:
We don't use documentation now, and don't have a 100% test case coverage.  We already ask for team members to describe tests above.  Should we remove these two items from the checklist?